libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -587,15 +595,15 @@ float SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::Aux_servo_functi
 /*
   get mask of output channels for a function
  */
-uint16_t SRV_Channels::get_output_channel_mask(SRV_Channel::Aux_servo_function_t function)
+uint32_t SRV_Channels::get_output_channel_mask(SRV_Channel::Aux_servo_function_t function)
 {
     if (!initialised) {
         update_aux_servo_function();
     }
     if (SRV_Channel::valid_function(function)) {
         return functions[function].channel_mask;
     }
-    return 0;
+    return invalid_mask;
```

Comment:
What's this patch for?

Reply:
it is needed for this line:
const uint32_t enabled_mask = ~SRV_Channels::get_output_channel_mask(SRV_Channel::k_GPIO);
in AP_Logger::Write_RCOUT()